src/main/java/nl/bo/techiteasydeel1/dtos/TelevisionDto.java

```diff
@@ -1,24 +1,45 @@
 package nl.bo.techiteasydeel1.dtos;
 
+import java.util.List;
+
 public class TelevisionDto {
-    String type;
-    String brand;
-    String name;
-    Double price;
-    Double availableSize;
-    Double refreshRate;
-    String screentype;
-    String screenQuality;
-    Boolean smartTv;
-    Boolean wifi;
-    Boolean voiceControl;
-    Boolean hdr;
-    Boolean bluetooth;
-    Boolean ambiLight;
-    Integer originalStock;
-    Integer sold;
+    public RemoteControllerDto remoteController;
```

Comment:
Het is prima om een DTO in een DTO te zetten, maar let op recursie. Als jou RemoteControllerDto nou ook weer een TelevisionDto bevat, dan krijg je een infinite loop.
Soms is het beter om hier niet een complete DTO, maar alleen een "Long remoteControllerId" te zetten.